lib/gui/models/supported-formats.js

```diff
@@ -115,7 +115,7 @@ SupportedFormats.service('SupportedFormatsModel', function() {
    * }
    */
   this.isSupportedImage = (imagePath) => {
-    const extension = path.extname(imagePath).slice(1).toLowerCase();
+    const extension = _.join(_.tail(path.extname(imagePath)), '').toLowerCase();
```

Comment:
Sounds good. I used `_.replace`, but decided to not store another variable with the extension + the period, since its tricky to find a good name for each, so that its not confusing.